packages/cli/src/commands/serve.js

```diff
@@ -206,9 +206,9 @@ export const builder = async (yargs) => {
 }
 
 // We'll clean this up later, but for now note that this function is
-// duplicated between this package and @redwoodjs/fastify
-// to avoid importing @redwoodjs/fastify when the CLI starts.
-export function coerceRootPath(path) {
+// duplicated between this package and @redwoodjs/fastify-web
+// to avoid importing @redwoodjs/fastify-web when the CLI starts.
+function coerceRootPath(path) {
```

Comment:
Not exported anymore, to make it clear that no one us importing this function from anywhere